esp/src/eclwatch/UserQueryWidget.js

```diff
@@ -157,6 +157,7 @@ define([
         _onCheckFilePermissions: function () {
             var context = this;
             this.filePermissionDialog.show();
+            this.filePermissionsForm.reset();
```

Comment:
This be done onClose.

Reply:
i submitted it backwards, but when set on close, it only resets when the user clicks the close button, however if the click "x" on the dialog box that function doesnt clear the dialog. this ensure that the form is always cleared.

Reply:
Yes, but the data appears and then just disappears on open. Here is the API for the dijit/Dialog for all cancel,hide,close options. https://dojotoolkit.org/api/#1_10dijit_Dialog_onCancel